cmd/podman/compose.go

```diff
@@ -40,6 +40,10 @@ If you want to change the default behavior or have a custom installation path fo
 	Annotations:        map[string]string{registry.ParentNSRequired: ""}, // don't join user NS for SSH to work correctly
 }
 
+const (
+	pipePrefix = "npipe:////./pipe/"
```

Comment:
given this is only used once, should it just be inline or is having the const defined worth it.